crates/codegen/runtime/generator/src/bindings/mod.rs

```diff
@@ -11,6 +11,10 @@ pub struct BindingsModel {
 impl BindingsModel {
     pub fn from_language(language: &model::Language) -> Result<Self> {
         let binding_rules_source = language.binding_rules_file.read_to_string()?;
```

Comment:
I suggest using `CodegenFileSystem` which will manage `cargo:rerun-if-changed` and a bunch of other system-wide checks for correctness/freshness.

Reply:
Sounds good. I'll change this.